susemanager-utils/susemanager-sls/salt/bootstrap/init.sls

```diff
@@ -21,6 +21,8 @@ no_ssh_push_key_authorized:
 {%- include 'channels/disablelocalrepos.sls' %}
 {% do repos_disabled.update({'skip': true}) %}
 
+{%- set transactional = grains['transactional'] %}
```

Comment:
Doesn't this throw an error on non-transactional systems?

Reply:
Tried it with non-transactional as well. And see the code https://github.com/saltstack/salt/blob/master/salt/grains/extra.py#L96

It just check existence of `transactional-update` script.